src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -607,23 +622,29 @@ void generateCdfJson(List<CastVoteRecord> castVoteRecords) throws IOException {
             entry("CVRContestSelection", selectionMapList),
             entry("@type", "CVR.CVRContest"));
 
-    Map<String, Object> snapshotMap =
-        Map.ofEntries(
+    return Map.ofEntries(
             entry("@id", generateCvrSnapshotID(cvr.getID(), round)),
             entry("CVRContest", new Map[]{contestMap}),
             entry("Type", round != null ? "interpreted" : "original"),
             entry("@type", "CVR.CVRSnapshot"));
-
-    return snapshotMap;
   }
 
   private Map<String, Object> generateCdfMapForElection() {
     HashMap<String, Object> electionMap = new HashMap<>();
 
-    List<Map<String, String>> contestSelections = new LinkedList<>();
-    for (String candidate : config.getCandidateCodeList()) {
-      contestSelections.add(
-          Map.ofEntries(entry("@id", candidate), entry("@type", "CVR.ContestSelection")));
+    List<Map<String, Object>> contestSelections = new LinkedList<>();
+    for (String candidateCode : config.getCandidateCodeList()) {
+      Map<String, String> codeMap = Map.ofEntries(
+          entry("@type", "CVR.Code"),
+          entry("Type", "other"),
+          entry("OtherType", "vendor-label"),
```

Comment:
vendor-label doesn't seem right

Reply:
I'm just following what John suggested in his email.

Reply:
In that case, great :)